fgd/point/env/env_outtro_stats.fgd

```diff
@@ -1,4 +1,5 @@
-@PointClass base(BaseEntityPoint) 
+@PointClass appliesto(+USE_MULTIPLAYER)
```

Comment:
I don't believe this is a multiplayer entity

Reply:
> I don't believe this is a multiplayer entity

It's from l4d, it controls the end of campaign stats, so I guess you're right. Would I give -MOMENTUM?

Reply:
Good to remove this entirely. We'll rip it out of game code because it is really l4d specific.

Reply:
Alright